packages/embark-ui/src/services/web3.js

```diff
@@ -30,3 +30,18 @@ export function deploy({web3, contract, args}) {
       .then(() => {});
     });
 }
+
+export function isDeployed({web3, contract}) {
+  if(!contract.deployedAddress) return Promise.resolve(false);
+  return new Promise((resolve, reject) => {
+    web3.eth.getCode(contract.deployedAddress, (err, byteCode) => {
+      const deployedAddress = contract.deployedAddress.replace("0x", "").toLowerCase();
+      if(err) return reject(err);
```

Comment:
I would put this above the other line, as `deployedAddress` is not used in case of failure

Reply:
Updated, thank you!